src/components/AppLayout/index.tsx

```diff
@@ -1200,7 +1200,7 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 						</div>
 					</Layout>
 				) : (
-					<Layout className={'min-h-[calc(100vh - 10rem)] flex flex-row bg-[#F5F6F8] dark:bg-section-dark-background'}>
+					<Layout className={'min-h-[calc(100vh - 10rem)] overflow-x-none overflow-y-none flex flex-row bg-[#F5F6F8] dark:bg-section-dark-background'}>
```

Comment:
overflow none?

Reply:
woh cards swipe k time x and y direction se space leraha tha so overflow band kr diya